### ISSUE TITLE
Inconsistent indents depending on where object literal open bracket is placed.

### ISSUE THREAD

**Issue by q42jaap** [eslint/eslint#5097]
ESLint: `eslint@1.10.3`

```javascript
findAndModify({
  query: {_id},
  update: updateDoc,
  upsert: true,
  new: false,
})
  .then(result => result.value);
```
my indent settings are `"indent": [1, 2, {"SwitchCase": 1}],`

I would expect the contents of the first parameter to findAndModify not to align with the `.then` continuation.

Because, when an argument is added:
```javascript
findAndModify(thing,
  {
    query: {_id},
    update: updateDoc,
    upsert: true,
    new: false,
  })
  .then(result => result.value);
```
Then the object literal shifts an indentation. Imho, the choice of where the opening curly is placed should not influence where a closing `})` is located. In the first case, `})` is at indent 0. In the second case, just because the starting `{` is on a new line, the closing `})` is at indent 2.

This makes the code look inconsistent.

**Comment by eslintbot** [comment]
@q42jaap Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
I'm not sure what it is you're reporting. Can you include the ESLint output and explain what it is about the output that is unexpected?

**Comment by ilyavolodin** [comment]
Closing as a duplicate of #1801 Feel free to reopen if you don't agree.
